### ISSUE TITLE
min-width with a non-px unit and either padding or border causes IE6 to hang

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#56]
```
What steps will reproduce the problem?
1. Open the attached test case in Internet Explorer 6.

There's a <div class="badclass"> and .badclass { min-height: 20em; padding:
1px } in a <style> section in the <head>. If I change the "em" to "px",
it's fine. If I remove the padding, it's fine.

But a non-px unit (eg, "in" or "pt" will also cause this to happen)
combined with one or both of a border or padding causes IE6 to lock up and
have to be killed.

What version of the product are you using? On what operating system?

IE7.js 2.0(beta3) on Internet Explorer 6 under Windows XP. IE6 says in Help
> About that its version is "6.0.2900.2180.xpsp_sp2_qfe.070227-2300"
```

Original issue reported on code.google.com by `jcre...@gmail.com` on 14 Mar 2008 at 3:31

Attachments:
- [ie7test.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-56/comment-0/ie7test.html)

**Comment by GoogleCodeExporter** [comment]
```
Oops, I said "min-width" in the title. I meant to say "min-height". I did not 
test
with min-width, so I don't know if the bug applies there or not.
```

Original comment by `jcre...@gmail.com` on 14 Mar 2008 at 3:43

**Comment by GoogleCodeExporter** [comment]
```
confirmed, min-height with an em value causes ie6 to crash.
```

Original comment by `jaysw...@gmail.com` on 18 May 2009 at 5:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 3:02
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 13 Feb 2010 at 11:59
- Changed state: **Fixed**
